Packs/InfobloxBloxOne/Integrations/InfobloxBloxOneThreatDefense/InfobloxBloxOneThreatDefense.yml

```diff
@@ -106,11 +210,1758 @@ script:
+    - name: user
+      required: false
+      description: Specify the user of the assets.
+    description: List assets for a specific insight.
+    name: infobloxcloud-insight-assets-list
+    outputs:
+    - contextPath: InfobloxCloud.Asset.count
+      description: The number of occurrences associated with the asset.
+      type: Number
+    - contextPath: InfobloxCloud.Asset.qip
+      description: The IP address of the asset.
+      type: String
+    - contextPath: InfobloxCloud.Asset.location
+      description: The geographical location of the asset.
+      type: String
+    - contextPath: InfobloxCloud.Asset.threatLevelMax
+      description: The maximum threat level associated with the asset.
+      type: String
+    - contextPath: InfobloxCloud.Asset.threatIndicatorDistinctCount
+      description: The number of distinct threat indicators associated with the asset.
+      type: String
+    - contextPath: InfobloxCloud.Asset.timeMax
+      description: The latest time the asset was observed.
+      type: Date
+    - contextPath: InfobloxCloud.Asset.timeMin
+      description: The earliest time the asset was observed.
+      type: Date
+    - contextPath: InfobloxCloud.Asset.mostRecentAction
+      description: The most recent action taken for the asset.
+      type: String
+  - arguments:
+    - name: insight_id
+      required: true
+      description: Specify the insight ID to fetch comments for.
+      default: true
+    - name: start_time
+      required: false
+      description: "Specify the start time for the comments.\n\nFormat: YYYY-MM-DDTHH:MM:SSZ, YYYY-MM-DD, N days, N hours.\n\nExample: 2025-04-25T00:00:00Z, 2025-04-25, 2 days, 5 hours, 01 Mar 2025, 01 Feb 2025 04:45:33, 15 Jun."
+    - name: end_time
+      required: false
+      description: "Specify the end time for the comments.\n\nFormat: YYYY-MM-DDTHH:MM:SSZ, YYYY-MM-DD, N days, N hours.\n\nExample: 2025-04-25T00:00:00Z, 2025-04-25, 2 days, 5 hours, 01 Mar 2025, 01 Feb 2025 04:45:33, 15 Jun."
+    - name: limit
+      required: false
+      description: Specify the maximum number of comments to fetch.
+      defaultValue: 50
+    description: List comments for a specific insight.
+    name: infobloxcloud-insight-comments-list
+    outputs:
+    - contextPath: InfobloxCloud.Comment.commentsChanger
+      description: The user who created or changed the comment.
+      type: String
+    - contextPath: InfobloxCloud.Comment.dateChanged
+      description: The timestamp when the comment was created or modified.
+      type: Date
+    - contextPath: InfobloxCloud.Comment.status
+      description: The status associated with the comment.
+      type: String
+    - contextPath: InfobloxCloud.Comment.newComment
+      description: The comment text.
+      type: String
   runonce: false
   script: '-'
   type: python
   subtype: python3
   dockerimage: demisto/python3:3.12.11.4508456
+  isfetch: true
 fromversion: 6.5.0
```

Comment:
the new content you added has the fromversion  set to 6.10 while the integration is 6.5, please change theirs to be 6.5